inject/android.js

```diff
@@ -16,10 +25,43 @@ function initCode () {
         platform: processedConfig.platform
     })
 
+    const messageSecret = processedConfig.messageSecret
+    const messageCallback = processedConfig.messageCallback
```

Comment:
Nit: Could you add comments for `messageCallback` and `messageInterface` to make it clear which is for sending messages to the Android code and which is for receiving messages from the Android code?